### ISSUE TITLE
 get_data_by_command cause garbage

### ISSUE THREAD

**Issue by yoonje** [iipeace/guider#221]
get_data_by_command(target_addr, request_id, cmd) cause garbage result.

I checked str_pipe = pipe.getData() return correctly.

But in my code print(result) using str_pipe return garbage.

garbage data form : {'data': '\n', 'errMsg': '', 'result': 0}

```py
def get_data_by_command(target_addr, request_id, cmd):
    result = dict(result=0, data=dict(), errMsg='')
    pipe = None
    if cmd is None or cmd is '' or target_addr is None or target_addr is '':
        result['result'] = -1
        result['errorMsg'] = 'Required data missing'
        sys.exit(0)
    try:
        RequestManager.add_request(request_id)
        GuiderInstance.set_network_manager(target_addr)
        pipe = GuiderInstance.get_command_pipe(target_addr, cmd)
        while RequestManager.get_request_status(request_id):
            str_pipe = pipe.getData()
            if not str_pipe:
                break
            result['data'] = str_pipe.replace('\n', '')
            print(result)
```

<img width="1288" alt="result" src="https://user-images.githubusercontent.com/38535571/82139967-fbfd7780-9866-11ea-8c33-abd579c07cdb.png">

**Comment by iipeace** [comment]
could you test again after pulling recent source?
and pipe.getDataType(output) will be very useful to check type for pipe data.

**Comment by yoonje** [comment]
It is the same when running with the code of the current master branch. 
If there is garbage data, it is implemented by exception handling.

**Comment by yoonje** [comment]
fixed from commit 6e825a340dde6f6f3b90b6e3ec10db05ab7334a8
